extensions/oaipmh/deployment/src/main/java/org/apache/camel/quarkus/component/oaipmh/deployment/OaipmhProcessor.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.quarkus.component.oaipmh.deployment;
+
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.deployment.builditem.ExtensionSslNativeSupportBuildItem;
+import io.quarkus.deployment.builditem.FeatureBuildItem;
+import io.quarkus.deployment.builditem.nativeimage.NativeImageResourceDirectoryBuildItem;
+
+class OaipmhProcessor {
+
+    private static final String FEATURE = "camel-oaipmh";
+
+    @BuildStep
+    FeatureBuildItem feature() {
+        return new FeatureBuildItem(FEATURE);
+    }
+
+    @BuildStep
+    ExtensionSslNativeSupportBuildItem activateSslNativeSupport() {
+        return new ExtensionSslNativeSupportBuildItem(FEATURE);
+    }
+
+    @BuildStep
+    NativeImageResourceDirectoryBuildItem registerNativeResources() {
+        return new NativeImageResourceDirectoryBuildItem("org/joda/time/tz/data");
```

Comment:
We should consider a separate support (or Quarkiverse) extension once we have more stuff for Joda time. It is OK, as far as it is just a single item. Shouldn't we at least file a "Consider separate Joda extension" issue?

Reply:
Filed https://github.com/apache/camel-quarkus/issues/2191.